docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_java_dependencies.adoc

```diff
@@ -183,7 +263,7 @@ kubectl annotate artifactbuild --all jvmbuildservice.io/rebuild=failed
 
 Generally, when you are trying to fix a failure, you must manually run the builds yourself.
 
-=== Dealing With Failed Builds
+== Dealing With failed Builds
 
 In order to see why the build failed, look at the results from the JVM Build Service.
```

Comment:
[no "in order to"](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i) (for minimalism). just say "to".